test/index.js

```diff
@@ -0,0 +1,17 @@
+var Lab    = require('lab')
+var assert = require('assert')
+var server = require('../lib/server')(8000)
+
+var lab = exports.lab = Lab.script()
+
+lab.test('/version endpoint returns the project version', function(done) {
+  server.inject('/version', function(response) {
+    var result = response.result
+
+    assert.ok(result instanceof Object)
+    assert.equal(response.statusCode, 200)
+    assert.equal(result.version, '0.0.1');
```

Comment:
To a degree, yes. I have difficulty providing an authoritive answer. Most of the behavior here relates to routing, so the duplication is okay by me. 

In my submission, I export the expected value from a server.js module, but you could also bring that up for debate as well.